### PR TITLE
feat: uikit 4.0 dynamic config parse logic

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/providers/config-provider.tsx [hunk 1]
```diff
@@ -10,20 +14,57 @@ export const ConfigContext = createContext<IUIKitConfig>({
...
+  };
+
+  const getCorrectConfig = (
+    correctIds: string[]
+  ): Record<string, string | string[] | Record<string, string>> => {
+    for (let i = 0; i < correctIds.length; i++) {
+      const correctId = correctIds[i];
+      const correctConfig = configs.customizations[correctId];
+      if (correctConfig) return correctConfig;
+    }
```
Comment (frankent) [inline src/providers/config-provider.tsx:48]: Can you use `Array.find` instead for this loop

**Review by frankent — CHANGES_REQUESTED** [review]
Please check my comment

File: src/providers/config-provider.tsx (hunk 1 above)
Comment (NaingAmity) [reply src/providers/config-provider.tsx:48]: We want to break the loop as soon as we found the element. 
I am not sure array.find will break if we find the specific element
What do you think?

Comment (NaingAmity) [reply src/providers/config-provider.tsx]: tested and it worked so I updated with Array,.find Thanks xD

**Review by chaiwattsw — APPROVED** [review]
LGTM

**Pull request merged**